### PR TITLE
fix(stats): first version of stats

### PR DESCRIPTION
**Pull request by WikiRik**
Couldn't figure out a nice way to do the TODO, so we'll fix that for next year

### CODE REVIEW

File: lib/applications.js [hunk 1]
```diff
@@ -347,3 +347,34 @@ exports.exportAll = async (req, res) => {
...
+        by_body: [],
+        by_nationality: []
+    };
+
+    const applications = await Application.findAll({ attributes: ['event_id', 'body_name', 'nationality'] });
+
+    statsObject.by_event = helpers
+        .countByField(applications, 'event_id');
+
+    for (let i = 0; i < statsObject.by_event.length; i++) {
```
Comment (serge1peshcoff) [inline lib/applications.js:363]: this would do quite a lot of requests, can optimize it here by preloading all events at once

Comment (WikiRik) [reply lib/applications.js]: Good one, I should have improved it now

**Review by serge1peshcoff — APPROVED** [review]
can you also add some tests to maintain 100% coverage? other than that lgtm

**Comment by WikiRik** [comment]
SU module does not have tests yet 🙈

But tests are on the backlog for next year's improvements

**Pull request merged**